### PR TITLE
Added custom container explainability notebook

### PR DESCRIPTION
**Pull request by brianchunkang**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb [hunk 1]
```diff
@@ -0,0 +1,1283 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>2021 -&gt; 2022 ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>links for Colab and Workbench</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>Vertex Prediction -&gt; Vertex AI Prediction</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>Use the notebook template cell for REGION</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>BUCKET_NAME -&gt; BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>end of what?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>Each code cell should have text cell explaining what it does</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>explain how it works</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>remove?</p><p>'''</p><p>....</p><p>'''</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>use notebook template cell for credentials</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>duplicate cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>more explanations of code cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Comment (andrewferlitsch) [inline notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>add</p><p> </p><p>delete_bucket = False</p><p>if delete_bucket or os.getenv("IS_TESTING"):</p><p>    ! gsutil rm -rf {BUCKET_URI}</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>added</p>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>removed</p>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>updated</p>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>This is the code for the prediction container</p>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>updated</p>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>updated</p>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>updated</p>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>done</p>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>updated</p>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>added</p>

Comment (brianchunkang) [reply notebooks/community/explainable_ai/SDK_Custom_Container_XAI.ipynb:1]: <p>updated</p>

**Pull request merged**